chaoscenter/authentication/pkg/utils/project_utils.go

```diff
@@ -0,0 +1,203 @@
+package utils
+
+import (
+	"log"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+	"github.com/litmuschaos/litmus/chaoscenter/authentication/pkg/entities"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+func GetProjectFilters(c *gin.Context) *entities.ListProjectRequest {
```

Comment:
Functions in this package can be updated by taking ref from: https://github.com/litmuschaos/litmus/blob/master/chaoscenter/graphql/server/pkg/chaos_experiment/handler/handler.go#L508

Reply:
I actually took that reference to make those functions, do you find anything wrong in there?